css/style.scss

```diff
@@ -921,65 +898,48 @@ video {
 /**
  * Sidebar details view
  */
-
-.detailCallInfoContainer .room-name {
-	display: inline-block;
-}
-
-.detailCallInfoContainer h3,
-.newCommentRow .guest-name p,
-.detailCallInfoContainer .guest-name p {
-	display: inline-block;
-}
-
-.newCommentRow .guest-name p,
-.detailCallInfoContainer .guest-name p {
-	padding: 9px 0;
-}
-
-.newCommentRow .editable-text-label .edit-button,
-.detailCallInfoContainer .editable-text-label .edit-button {
-	display: none;
-}
-
-.detailCallInfoContainer .clipboard-button,
-.detailCallInfoContainer .password-button,
-.newCommentRow .editable-text-label:hover .edit-button,
-.detailCallInfoContainer .editable-text-label:hover .edit-button {
-	display: inline-block;
-}
-
-.detailCallInfoContainer .clipboard-button .icon,
-.detailCallInfoContainer .password-button .icon,
-.newCommentRow .editable-text-label .edit-button .icon,
-.detailCallInfoContainer .editable-text-label .edit-button .icon {
-	cursor: pointer;
-	padding: 22px;
-	vertical-align: middle;
-	margin-left: -5px;
-	margin-top: -5px;
-}
-
-.newCommentRow .editable-text-label .input-wrapper,
-.detailCallInfoContainer .editable-text-label .input-wrapper,
-.detailCallInfoContainer .password-option {
-	position: relative;
-	display: inline-block;
-}
-
-.newCommentRow .editable-text-label input,
-.detailCallInfoContainer .editable-text-label input,
-.detailCallInfoContainer .password-input {
-	width: 100%;
-}
-
-.newCommentRow .editable-text-label .label {
-	margin-left: 5px;
```

Comment:
Can you bring this back, looks much better then, because name and chat input are in line